IncSet.c

```diff
@@ -155,6 +157,23 @@ static bool IncMode_find(const IncMode* mode, Panel* panel, IncMode_GetPanelValu
    }
 }
 
+static int IncSet_backspaceLen(const char *s, size_t slen) {
+   size_t i = 0;
+   int mblen = 0;
+
+   mbtowc(NULL, 0, 0);
+
+   while (i < slen) {
+      mblen = mbtowc(NULL, s + i, slen - i);
```

Comment:
Re-reviewing this after I have more knowledge with the htop codebase (especially the `RichString` part).

As `RichString.c` is already using `mbrtowc` for wide character conversion, it's better to stick with `mbrtowc` here rather than downgrade to `mbtowc`. With `mbrtowc`, the string length can be properly addressed as `size_t` type.

Reply:
Also. It's good to rename the local variable `mblen` to `mbbuf_len` or something to avoid name collision with [the standard library function](https://man7.org/linux/man-pages/man3/mblen.3.html). (Name collision is not an error in C, but it can produce noise when other programmers `grep` for the keyword in code.)

Reply:
Shadowing should also be avoided where possible, as this may introduce hard-to-notice issues.